README.md

```diff
@@ -10,13 +10,35 @@ The members of Full Stack Embedded 2016 are, in alphabetical order:
 
 * Frederic Afadjigla
 * Nico Caspari
+* William Herndon
 * Daniel Lee
-* Rüdiger Reiter
+* Ruediger Reiter
 * Gregor Schnee
 
 This component was authored mainly by [Daniel Lee](erget2005@gmail.com) and 
 [Gregor Schnee](schneegor@gmail.com).
 
+## Required/suggested packages apt-get packages
+Use "sudo apt-get install <package-name>[, <package-name>]"
+### Required:
+* python3
+* python3-pip (required for setup.py to run)
+
+## Suggested for development setup:
```

Comment:
Hi Daniel!

I'm not sure I'm following you, but sure, you can remove the
suggested stuff and say that a webserver is needed.

Contrarily, if Django really is needed, I should add it to configure, right?

Regards,
Bill


 

 

 

-----Original Message-----
From: Daniel Lee <notifications@github.com>
To: FullStackEmbedded/WeatherServer <WeatherServer@noreply.github.com>
Cc: William David Herndon <wdherndon@aol.com>
Sent: Sun, Sep 6, 2015 11:38 pm
Subject: Re: [WeatherServer] configure and setup.py, with zypper support (#8)


 
In README.md:
  
>  * Gregor Schnee
>  
>  This component was authored mainly by [Daniel Lee](erget2005@gmail.com) and 
>  [Gregor Schnee](schneegor@gmail.com).
>  
> +## Required/suggested packages apt-get packages
> +Use "sudo apt-get install <package-name>[, <package-name>]"
> +### Required:
> +* python3
> +* python3-pip (required for setup.py to run)
> +
> +## Suggested for development setup:

  
I can see why you've included this, but I actually don't think that for a dev setup people will need these packages - especially since our dev server actually only requires python3 and django. When used in a production setting, apache is a great web server, but there are plenty of other ones out there. The same thing goes for mysql. I'll pull them in for now, but before merging into master I'll probably remove them while referring to the documentation from Django to avoid redundancy. Simultaneously, I'll be improving the documentation for setting up the dev server. Is that okay for you?
  
—
Reply to this email directly or view it on GitHub.